packages/mirage-flow-lwt/mirage-flow-lwt.1.2.0/opam

```diff
@@ -17,7 +17,8 @@ depends: [
   "topkg" {build & >= "0.7.3"}
   "fmt"
   "lwt"
-  "cstruct" {>= "2.0.0"}
+  ("cstruct" {>= "2.0.0" & < "3.0.0"}
+  | ("cstruct" {>= "3.0.0"} & "cstruct-lwt" {>= "3.0.0"}))
```

Comment:
this should just be adding a `cstruct-lwt` dependency -- the constraints on those packages take care of the version of cstruct